<!DOCTYPE html><html><head><meta charSet="utf-8"/><meta name="viewport" content="width=device-width,initial-scale=1"/><meta name="google-site-verification" content="c8xLy6CbRSnA2A3H8CUama6rq8hjtG1c_lBvFhO_nYs"/><meta name="next-head-count" content="3"/><link rel="preload" href="/ISBRA/_next/static/css/ad03d55cb9cb1936.css" as="style"/><link rel="stylesheet" href="/ISBRA/_next/static/css/ad03d55cb9cb1936.css" data-n-g=""/><link rel="preload" href="/ISBRA/_next/static/css/b14e9093fe1c298a.css" as="style"/><link rel="stylesheet" href="/ISBRA/_next/static/css/b14e9093fe1c298a.css" data-n-p=""/><noscript data-n-css=""></noscript><script defer="" nomodule="" src="/ISBRA/_next/static/chunks/polyfills-5cd94c89d3acac5f.js"></script><script src="/ISBRA/_next/static/chunks/webpack-6f45cb54dcc402af.js" defer=""></script><script src="/ISBRA/_next/static/chunks/framework-a87821de553db91d.js" defer=""></script><script src="/ISBRA/_next/static/chunks/main-803fd245650d2bfd.js" defer=""></script><script src="/ISBRA/_next/static/chunks/pages/_app-f19239920bd0d944.js" defer=""></script><script src="/ISBRA/_next/static/chunks/pages/agenda-62fda32ae0282319.js" defer=""></script><script src="/ISBRA/_next/static/lCx8W2dIrIX8zJLeGMmLd/_buildManifest.js" defer=""></script><script src="/ISBRA/_next/static/lCx8W2dIrIX8zJLeGMmLd/_ssgManifest.js" defer=""></script><script src="/ISBRA/_next/static/lCx8W2dIrIX8zJLeGMmLd/_middlewareManifest.js" defer=""></script></head><body><div id="__next"><header class="Layout_header__KD6xC"><div><h2>ISBRA 2022</h2></div></header><section class="Layout_hero__dJiuI"><div><h2>18th International Symposium on Bioinformatics Research and Applications (ISBRA 2022) November 14 - 17, 2022<!-- --><div>University of Haifa</div></h2></div></section><div class="Layout_root__XpAuI"><nav class="Layout_menu__N__qO"><h2>Menu</h2><ul><li>Home</li><li>Important Dates</li><li>Registration and Venue</li><li>Program Agenda</li><li>Submission Guidelines</li><li>Keynote Speakers</li><li>Call for Papers / Abstracts</li><li>Committees</li><li>Previous Meetings</li></ul></nav><div class="Layout_stripe__65gNv"></div><main class="Layout_main__gbdP0"><h1>Program agenda</h1><div class="general_notice__4QaUE"><a class="general_link__OzBkF" rel="noreferrer" target="_blank" href="https://drive.google.com/file/d/1KhirD4vE5u7A2cJLCvn2ZFUQ-A0pF0WC/view?usp=sharing">ISBRA 2022 Program (preliminary draft)</a></div><h1>Monday, November 14th, 2022</h1><table><thead><tr><td>7:00pm-<!-- --><br/>9:00pm<!-- --></td><td><b>Reception</b></td></tr></thead></table><p></p><h1>Tuesday, November 15th, 2022</h1><table><thead><tr><th>9:00am-<!-- --><br/>10:00am<!-- --></th><th>Plenary Session Room: TBA<!-- --><br/>Chair: TBA<!-- --></th></tr></thead><tbody><tr><td></td><td>Session1:<!-- --><br/>Keynote Talk<!-- --><br/>Eugene Koonin, National Center for Biotechnology Information,<!-- --><br/>National Institutes of Health<!-- --></td></tr><tr><td>10:00am-<!-- --><br/>10:20am<!-- --></td><td>Coffee break</td></tr></tbody></table><p></p><table><thead><tr><th>10:20am-<!-- --><br/>12:20pm<!-- --></th><th colspan="2">Parallel Sessions</th></tr></thead><tbody><tr><td></td><td>Session 2A:<!-- --><br/>Evolution, Ecology, Comparative<!-- --><br/>Genomics<!-- --><br/>Room:<!-- --><br/>Chair:<!-- --><br/></td><td>Session 2B:<!-- --><br/>Machine Learning: Medical<!-- --><br/>Records/Data, Protein Domains<!-- --><br/>Room:<!-- --><br/>Chair:<!-- --><br/></td></tr><tr><td>10:20am-<!-- --><br/>10:40am<!-- --></td><td>BLASTphylo - an Interactive Web Tool<!-- --><br/>for Taxonomic and Phylogenetic<!-- --><br/>Analysis of Genes, Susanne Zabel,<!-- --><br/>Jennifer Müller, Friedrich Götz and KayNieselt<!-- --></td><td>Automatic ICD Coding based on Multi-<!-- --><br/>granularity Feature Fusion, Ying Yu, Han<!-- --><br/>Jiang, Junwen Duan and Jianxin Wang<!-- --></td></tr><tr><td>10:40am-<!-- --><br/>11:00am<!-- --></td><td>A Clonal Evolution Simulator for<!-- --><br/>Planning Somatic Evolution Studies,<!-- --><br/>Arjun Srivatsa, Haoyun Lei and Russell<!-- --><br/>Schwartz<!-- --></td><td>Fusing Label Relations for Chinese EMR<!-- --><br/>Named Entity Recognition with Machine<!-- --><br/>Reading Comprehension, Shuyue Liu,<!-- --><br/>Junwen Duan, Feng Gong, Hailin Yueand Jianxin Wang<!-- --></td></tr><tr><td>11:00am-<!-- --><br/>11:20am<!-- --></td><td>Transposition Distance Considering<!-- --><br/>Intergenic Regions for Unbalanced<!-- --><br/>Genomes, Alexsandro Oliveira Alexandrino, Andre Rodrigues Oliveira,Geraldine Jean, Guillaume Fertin,Ulisses Dias and Zanoni Dias<!-- --></td><td>Effectively Training MRI Reconstruction<!-- --><br/>Network via Sequentially Using<!-- --><br/>Undersampled k-Space Data with Very Low Frequency Gaps, Tian-Yi Xing,<!-- --><br/>Xiao-Xin Li, Zhi-Jie Chen, Xi-Yu Zhengand Fan Zhang<!-- --></td></tr><tr><td>11:20am-<!-- --><br/>11:40am<!-- --></td><td>Interactions between culturable bacteria<!-- --><br/>are predicted by individual species’<!-- --><br/>growth, Jonathan Friedman and Einat<!-- --><br/>Nestor<!-- --></td><td>Research on the prediction method of<!-- --><br/>disease classification based on imaging<!-- --><br/>features, Yu Sheng, Shengyi Yang,<!-- --><br/>Huirong Hu and Guihua Duan<!-- --></td></tr><tr><td>11:40am-<!-- --><br/>12:00pm<!-- --></td><td>Using variable-order Markov models to<!-- --><br/>improve metagenomics binning and<!-- --><br/>genome taxonomic classification,<!-- --><br/>Leibovich Or, Yochai Meir and Itai<!-- --><br/>Sharon<!-- --></td><td>NIDN: Medical Code Assignment via<!-- --><br/>Note-Code Interaction Denoising<!-- --><br/>Network, Xiaobo Li, Yijia Zhang,<!-- --><br/>Xingwang Li, Jian Wang and Mingyu Lu<!-- --></td></tr><tr><td>12:00pm-<!-- --><br/>12:20pm<!-- --></td><td>Revisiting SVDQuartets: Performance<!-- --><br/>under Real Life Values and<!-- --><br/>Improvements, Shay Hakim and Sagi<!-- --><br/>Snir<!-- --></td><td>GNN-Dom: an unsupervised method for<!-- --><br/>protein domain partition via protein<!-- --><br/>contact map, Wang Lei and Yan Wang<!-- --></td></tr><tr><td>12:20pm-<!-- --><br/>2:00pm<!-- --></td><td colspan="2">Lunch Break</td></tr></tbody></table><p></p><table><thead><tr><th>2:00pm-<!-- --><br/>3:00pm<!-- --></th><th>Plenary Session Room:<!-- --><br/>Chair:<!-- --></th></tr></thead><tbody><tr><td></td><td>Session 3:<!-- --><br/>Keynote Talk<!-- --><br/>Talk Title<!-- --><br/>Tamar Friedlander, Faculty of Agriculture,<!-- --><br/>Hebrew University of Jerusalem<!-- --></td></tr><tr><td>3:00pm-<!-- --><br/>3:20pm<!-- --></td><td>Coffee Break</td></tr></tbody></table><p></p><table><thead><tr><th>3:20pm-<!-- --><br/>5:40pm<!-- --></th><th colspan="2">Parallel Sessions</th></tr></thead><tbody><tr><td></td><td>Session 4A:<!-- --><br/>Cancer, SARS-CoV-2<!-- --><br/>Room:<!-- --><br/>Chair:<!-- --></td><td>Session 4B:<!-- --><br/>Networks, Machine Learning<!-- --><br/>Room:<!-- --><br/>Chair:<!-- --></td></tr><tr><td>3:20pm-<!-- --><br/>3:40pm<!-- --></td><td>Tumor type-specific survival associations<!-- --><br/>of transcriptomic-weighted codon usage<!-- --><br/>and tRNA abundance, Douglas Meyer,<!-- --><br/>Haim Bar, Anton Komar, Ryan Hunt,<!-- --><br/>Michael DiCuccio and Chava Kimchi-<!-- --><br/>Sarfaty<!-- --></td><td>STgcor: A Distribution-based Correlation<!-- --><br/>Measurement Method for Spatial<!-- --><br/>Transcriptome Data, Xiaoshu Zhu,<!-- --><br/>Liyuan Pang, Wei Lan, Shuang Meng<!-- --><br/>and Xiaoqing Peng<!-- --></td></tr><tr><td>3:40pm-<!-- --><br/>4:00pm<!-- --></td><td>A Network-Based Voting Method for<!-- --><br/>Identification and Prioritization of<!-- --><br/>Personalized Cancer Driver Genes, Han<!-- --><br/>Li, Feng Li, Junliang Shang, Xikui Liu<!-- --><br/>and Yan Li<!-- --></td><td>An SMT-based Framework for<!-- --><br/>Reasoning about Discrete Biological<!-- --><br/>Models, Boyan Yordanov, Sara-Jane<!-- --><br/>Dunn, Colin Gravill, Hillel Kugler and<!-- --><br/>Christoph M. Wintersteiger<!-- --></td></tr><tr><td>4:00pm-<!-- --><br/>4:20pm<!-- --></td><td>A Tensor Robust Model Based on<!-- --><br/>Enhanced Tensor Nuclear Norm and<!-- --><br/>Low-Rank Constraint for Multi-view<!-- --><br/>Cancer Genomics Data, Qian Qiao, Sha-<!-- --><br/>Sha Yuan, J. Shang and Jin-Xing Liu<!-- --></td><td>Power and limitations of hybrid system<!-- --><br/>models for gene regulatory networks,<!-- --><br/>Juris Viksna, Kārlis Čerāns, Lelde Lace<!-- --><br/>and Gatis Melkus<!-- --></td></tr><tr><td>4:20pm-<!-- --><br/>4:40pm<!-- --></td><td>Efficient Distance-Based Reconstruction<!-- --><br/>of Large-Scale Single-Cell Phylogenies<!-- --><br/>from Copy Number Data, Samson<!-- --><br/>Weiner and Mukul S. Bansal<!-- --></td><td>Graph Random Forest: A graph<!-- --><br/>embedded algorithm for identifying<!-- --><br/>highly connected important features, Leqi<!-- --><br/>Tian and Tianwei Yu<!-- --></td></tr><tr><td>4:40pm-<!-- --><br/>5:00pm<!-- --></td><td>Evaluating the Robustness of ML Models<!-- --><br/>in SARS-CoV-2 Genome Sequences<!-- --><br/>Generated Using TGS Technology,<!-- --><br/>Bikram Sahoo, Sarwan Ali, Pin-Yu Chen,<!-- --><br/>Murray Patterson and Alex Zelikovsky<!-- --></td><td>Simulating Spiking Neural Networks<!-- --><br/>based on SW26010pro, Zhichao Wang,<!-- --><br/>Xuelei Li, Yi Pan and Yanjie Wei<!-- --></td></tr><tr><td>5:00pm-<!-- --><br/>5:20pm<!-- --></td><td>t-SNE Highlights Phylogenetic and<!-- --><br/>Temporal Patterns of SARS-CoV-2<!-- --><br/>Spike and Nucleocapsid Protein<!-- --><br/>Evolution, Gaik Tamazian, Andrey<!-- --><br/>Komissarov, Dmitry Kobak, Dmitry<!-- --><br/>Polyakov, Evgeny Andronov, Sergei<!-- --><br/>Nechaev, Sergey Kryzhevich, Yuri<!-- --><br/>Porozov and Eugene Stepanov<!-- --></td><td>A methodology for classifying tissue-<!-- --><br/>specific metabolic and inflammatory<!-- --><br/>receptor functions applied to<!-- --><br/>subcutaneous and visceral adipose,<!-- --><br/>Judith Somekh and Gur Arye<!-- --></td></tr><tr><td>5:20pm-<!-- --><br/>5:40pm<!-- --></td><td>A rigorous benchmarking of methods for<!-- --><br/>SARS-CoV-2 lineage quantification in<!-- --><br/>wastewater, Viorel Munteanu, Nitesh<!-- --><br/>Kumar Sharma, Bohdan Tyshchenko,<!-- --><br/>Alina Frolova, Serghei Knyazev and<!-- --><br/>Serghei Mangul<!-- --></td><td>Deep learning-enhanced MHC-II<!-- --><br/>presentation prediction and peptidome<!-- --><br/>deconvolution, Juntao Deng and Min Liu<!-- --></td></tr><tr><td>6:00pm-<!-- --><br/>8:00pm<!-- --></td><td colspan="2">Banquet</td></tr></tbody></table><h1>Wednesday, November 16th, 2022</h1><table><thead><tr><th>9:00am-<!-- --><br/>10:00am<!-- --></th><th>Plenary Session Room: TBA<!-- --><br/>Chair: TBA<!-- --></th></tr></thead><tbody><tr><td></td><td>Session 5:<!-- --><br/>Keynote Talk<!-- --><br/>Talk Title<!-- --><br/>Mona Singh, Professor of Computer Science,<!-- --><br/>Princeton University<!-- --></td></tr><tr><td>10:00am-<!-- --><br/>10:20am<!-- --></td><td>Coffee Break</td></tr></tbody></table><p></p><table><thead><tr><th>10:20am-<!-- --><br/>12:20pm<!-- --></th><th colspan="2">Parallel Sessions</th></tr></thead><tbody><tr><td></td><td>Session 5A:<!-- --><br/>Clustering, Sequence Analysis<!-- --><br/>Room:<!-- --><br/>Chair:<!-- --></td><td>Session 5B:<!-- --><br/>Privacy, PPI, Pathways<!-- --><br/>Room:<!-- --><br/>Chair:<!-- --></td></tr><tr><td>10:20am-<!-- --><br/>10:40am<!-- --></td><td>Entropy Based Clustering of Viral<!-- --><br/>Sequences, Akshay Juyal, Mark<!-- --><br/>Grinshpon, Daniel Novikov, Roya<!-- --><br/>Hosseini and Alex Zelikovsky<!-- --></td><td>Private Epigenetic PaceMaker Detector<!-- --><br/>using Homomorphic Encryption, Meir<!-- --><br/>Goldenberg, Adi Akavia and Sagi Snir<!-- --></td></tr><tr><td>10:40am-<!-- --><br/>11:00am<!-- --></td><td>FEED: a feature extraction method based<!-- --><br/>on gene expression decomposition for<!-- --><br/>single-cell clustering, Chao Zhang,<!-- --><br/>Yunpei Xu, Lu He, Tao Zhang and<!-- --><br/>Hongdong Li<!-- --></td><td>MPCDDI: A Secure Multiparty<!-- --><br/>Computation-based Deep Learning<!-- --><br/>Framework for Drug-drug Interaction<!-- --><br/>Predictions, Xia Xiao, Shaoliang Peng,<!-- --><br/>Shengyun Liu and Xiaoqi Wang<!-- --></td></tr><tr><td>11:00am-<!-- --><br/>11:20am<!-- --></td><td>ARGLRR: An Adjusted Random Walk<!-- --><br/>Graph Regularization Sparse Low-rank<!-- --><br/>Representation Method for Single-cell<!-- --><br/>RNA-sequencing Data Clustering, Zhen-<!-- --><br/>Chang Wang, Jin-Xing Liu, Jun-Liang<!-- --><br/>Shang, Ling-Yun Dai, Chun-Hou Zheng<!-- --><br/>and Juan Wang<!-- --></td><td>EMRShareChain: A Privacy-Preserving<!-- --><br/>EMR Sharing System Model Based on<!-- --><br/>the Consortium Blockchain, Xinglong<!-- --><br/>Zhang, Peng Xi, Wenjuan Liu and<!-- --><br/>Shaoliang Peng<!-- --></td></tr><tr><td>11:20am-<!-- --><br/>11:40pm<!-- --></td><td>An Efficient and User-friendly Software<!-- --><br/>for PCR Primer Design for Detection of<!-- --><br/>Highly Variable Bacteria, Dongzheng<!-- --><br/>Hu, Wubin Qu, Fan Tong, Xiangwen<!-- --><br/>Zheng, Jiangyu Li and Dongsheng Zhao<!-- --></td><td>Estimating enzyme expression and<!-- --><br/>metabolic pathway activity in mice,<!-- --><br/>Filipp Rondel, Roya Hosseini, Hafsa<!-- --><br/>Farooq and Alex Zelikovsky<!-- --></td></tr><tr><td>11:40am-<!-- --><br/>12:00pm<!-- --></td><td>Optimal sequence alignment to ED-<!-- --><br/>strings, Njagi Moses Mwaniki and Nadia<!-- --><br/>Pisanti<!-- --></td><td>Heterogeneous PPI network<!-- --><br/>representation learning for protein<!-- --><br/>complex identification, Peixuan Zhou,<!-- --><br/>Yijia Zhang, Fei Chen, Kuo Pang and<!-- --><br/>Mingyu Lu<!-- --></td></tr><tr><td>12:00pm-<!-- --><br/>12:20pm<!-- --></td><td>An Algorithm to Calculate the p-value of<!-- --><br/>the Monge-Elkan Distance, Petr Ryšavý<!-- --><br/>and Filip Železný<!-- --></td><td>Gaussian-enhanced Representation<!-- --><br/>Model for Extracting Protein-Protein<!-- --><br/>Interactions Affected by Mutations, Da<!-- --><br/>Liu, Yijia Zhang, Ming Yang, Fei<!-- --><br/>Chen and Mingyu Lu<!-- --></td></tr><tr><td>12:20pm-<!-- --><br/>2:00pm<!-- --></td><td colspan="2">Lunch Break</td></tr></tbody></table><p></p><table><thead><tr><th>2:00pm-<!-- --><br/>3:00pm<!-- --></th><th>Plenary Session Room:<!-- --><br/>Chair:<!-- --></th></tr></thead><tbody><tr><td></td><td>Session 6:<!-- --><br/>Keynote Talk<!-- --><br/>Talk Title<!-- --><br/>Michal Linial, Professor of Biochemistry and Bioinformatics,<!-- --><br/>Hebrew University of Jerusalem<!-- --></td></tr><tr><td>3:00pm-<!-- --><br/>3:20pm<!-- --></td><td>Coffee Break</td></tr></tbody></table><p></p><table><thead><tr><th>3:20pm-<!-- --><br/>5:20pm<!-- --></th><th colspan="2">Parallel Sessions</th></tr></thead><tbody><tr><td></td><td>Session 7A:<!-- --><br/>Human Disease 1<!-- --><br/>Room:<!-- --><br/>Chair:<!-- --></td><td>Session 7B:<!-- --><br/>RNA, Viral Transmission/Genomics<!-- --><br/>Room:<!-- --><br/>Chair:<!-- --></td></tr><tr><td>3:20pm-<!-- --><br/>3:40pm<!-- --></td><td>A multi-view perspective on<!-- --><br/>predicting human diseases based on<!-- --><br/>gut microbiome features, Efrat Muller,<!-- --><br/>Itamar Shiryan and Elhanan Borenstein<!-- --></td><td>Predict CircRNA-disease Associations<!-- --><br/>Based on Random Walk with Restart,<!-- --><br/>Principal Component Analysis and<!-- --><br/>Random Forest, Qiang Luo and Minzhu<!-- --><br/>Xie<!-- --></td></tr><tr><td>3:40pm-<!-- --><br/>4:00pm<!-- --></td><td>Multi-atlas Representations based on<!-- --><br/>Transformer for Autism Spectrum<!-- --><br/>Disorders Diagnosis, Jianchun Zhu, Jin<!-- --><br/>Liu, Junbin Mao, Yu Sheng and Yi Pan<!-- --></td><td>Distance Profiles of Optimal RNA<!-- --><br/>Foldings, Jinyi Liu, Isabel Duan, Santi<!-- --><br/>Santichaivekin and Ran Libeskind-Hadas<!-- --></td></tr><tr><td>4:00pm-<!-- --><br/>4:20pm<!-- --></td><td>M-US-EMRs: A Multi-Modal Data<!-- --><br/>Fusion Method of Ultrasonic Images and<!-- --><br/>Electronic Medical Records Used for<!-- --><br/>Screening of Coronary Heart Disease,<!-- --><br/>Bokai Yang, Yingnan Zuo, Shunxiang<!-- --><br/>Yang, G. Deng, S. Zhu and Yunpeng Cai<!-- --></td><td>A Locality-Constrained Linear Coding-<!-- --><br/>Based Ensemble Learning Framework<!-- --><br/>for Predicting Potentially Disease-<!-- --><br/>Associated MiRNAs, Yi Shen, Ying-Lian<!-- --><br/>Gao, Juan Wang, Bo-Xin Guan and Jin-<!-- --><br/>Xing Liu<!-- --></td></tr><tr><td>4:20pm-<!-- --><br/>4:40pm<!-- --></td><td>Machine Learning based biomarker<!-- --><br/>discovery for chronic kidney disease–<!-- --><br/>mineral and bone disorder (CKD-MBD),<!-- --><br/>Xiang Shao, Yukuan Lou, Weixin Kong,<!-- --><br/>Yutin Li, Qianpan Liu and Suhua Zhang<!-- --></td><td>Global profiling and annotation of<!-- --><br/>templated isomiRs dynamics across<!-- --><br/>Caenorhabditis elegans development,<!-- --><br/>Ganesh Panzade, Li Li, Shilpa Hebbar,<!-- --><br/>Isana Veksler-Lublinsky and Anna<!-- --><br/>Zinovyeva<!-- --></td></tr><tr><td>4:40pm-<!-- --><br/>5:00pm<!-- --></td><td>TDCOSR: A multimodality fusion<!-- --><br/>framework for association analysis<!-- --><br/>between genes and ROIs of Alzheimer’s<!-- --><br/>disease, Qi Zou, Yan Sun, Feng Li, Juan<!-- --><br/>Wang, Jin-Xing Liu and Junliang Shang<!-- --></td><td>Unlocking capacities of genomics for the<!-- --><br/>COVID-19 response and future<!-- --><br/>pandemics, Serghei Mangul and Alex<!-- --><br/>Zelikovsky<!-- --></td></tr><tr><td>5:00pm-<!-- --><br/>5:20pm<!-- --></td><td>Policy-based Hypertension Monitoring<!-- --><br/>using Formal Runtime Verification<!-- --><br/>Monitors, Abhinandan Panda, Srinivas<!-- --><br/>Pinisetty and Partha Roop<!-- --></td><td>Inference of viral transmission networks<!-- --><br/>under structural constraints, Kiril<!-- --><br/>Kuzmin, Sagi Snir, Yury Khudyakov and<!-- --><br/>Pavel Skums<!-- --></td></tr><tr><td>6:00pm-<!-- --><br/>8:00pm<!-- --></td><td colspan="2">ISBRA 2022 Organizer Dinner</td></tr></tbody></table><h1>Thursday, November 17th, 2022</h1><table><thead><tr><th>9:00am-<!-- --><br/>10:00am<!-- --></th><th>Plenary Session Room: TBA<!-- --><br/>Chair: TBA<!-- --></th></tr></thead><tbody><tr><td></td><td>Session 8:<!-- --><br/>Keynote Talk<!-- --><br/>Talk Title<!-- --><br/>Abraham B Korol, Professor, Institute of Evolution,<!-- --><br/>University of Haifa<!-- --></td></tr><tr><td>10:00am-<!-- --><br/>10:20am<!-- --></td><td>Coffee Break</td></tr></tbody></table><table><thead><tr><th>10:20am-<!-- --><br/>11:40am<!-- --></th><th colspan="2">Parallel Sessions</th></tr></thead><tbody><tr><td></td><td>Session 9A:<!-- --><br/>Human Disease 2<!-- --><br/>Room:<!-- --><br/>Chair:<!-- --></td><td>Session 9B:<!-- --><br/>Drug Interaction/Association<!-- --><br/>Room:<!-- --><br/>Chair:<!-- --></td></tr><tr><td>10:20am-<!-- --><br/>10:40am<!-- --></td><td>Applying machine learning and leave-<!-- --><br/>pair-out cross-validation to a clinical and<!-- --><br/>steroidomics dataset to find predictors for<!-- --><br/>preeclampsia, Alexander Biehl<!-- --></td><td>MLMVFE: A Machine Learning<!-- --><br/>Approach Based on Muli-View Features<!-- --><br/>Extraction for Drug-Disease Associations<!-- --><br/>Prediction, Ying Wang, Ying-Lian Gao,<!-- --><br/>Juan Wang, Junliang Shang and Jin-Xing<!-- --><br/>Liu<!-- --></td></tr><tr><td>10:40am-<!-- --><br/>11:00am<!-- --></td><td>MMLN: Leveraging Domain Knowledge<!-- --><br/>for Multimodal Diagnosis, Haodi Zhang,<!-- --><br/>Chenyu Xu, Peirou Liang, Ke Duan, Hao<!-- --><br/>Ren, Weibin Cheng and Kaishun Wu<!-- --></td><td>Mixed expert Model For Drug-Target<!-- --><br/>Interaction Prediction, Honghui Lian, Zhi<!-- --><br/>Zeng, Xiang Li, Huan Wang, Guodong Li<!-- --><br/>and Ziwen Cui<!-- --></td></tr><tr><td>11:00am-<!-- --><br/>11:20am<!-- --></td><td>Transfer Learning for Semi-Supervised<!-- --><br/>3D Left Atrium Segmentation through<!-- --><br/>Dual-Task Consistency Framework,<!-- --><br/>Jiahao Chen, Ruohan Lin, Xiaobin Lan,<!-- --><br/>Rongxing Wang, Xipeng Pan and Yanju<!-- --><br/>Zhang<!-- --></td><td>Prediction of Drug-disease Relationship<!-- --><br/>on Heterogeneous Networks Based on<!-- --><br/>Graph Convolution, Jiancheng Zhong,<!-- --><br/>Pan Cui, Zuohang Qu, Qiu Xiao and<!-- --><br/>Yihong Zhu<!-- --></td></tr><tr><td>11:20am-<!-- --><br/>11:40am<!-- --></td><td>2D Photogrammetry Image of<!-- --><br/>Adolescent Idiopathic Scoliosis<!-- --><br/>Screening Using Deep Learning, Zhenda<!-- --><br/>Xu, Jiazi Ouyang, Qiang Gao, Aiqian<!-- --><br/>Gan, Qihua Zhou, Jiahao Hu and Song<!-- --><br/>Guo<!-- --></td><td>A Multimodal Data Fusion-based Deep<!-- --><br/>Learning Approach for Drug-Drug<!-- --><br/>Interaction Prediction, An Huang,<!-- --><br/>Xiaolan Xie, Xiaoqi Wang and Shaoliang<!-- --><br/>Peng<!-- --></td></tr><tr><td>11:40am-<!-- --><br/>12:00pm<!-- --></td><td colspan="2">Closing Remarks</td></tr><tr><td>12:00 pm-<!-- --><br/>7:00pm<!-- --></td><td colspan="2">Lunch Break + Excursion</td></tr></tbody></table></main><section class="Layout_sponsors__UrytT"><h2>Sponsors &amp; Affiliates</h2><ul><li><a rel="noreferrer" target="_blank" href="https://csds.gsu.edu/">Georgia State University, Department of Computer Science</a></li><li><a rel="noreferrer" target="_blank" href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1923679&amp;HistoricalAwards=false">National Science Foundation, USA</a></li><li><a rel="noreferrer" target="_blank" href="https://www.iscb.org/">International Society for Computational Biology</a></li></ul></section></div></div><script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{}},"page":"/agenda","query":{},"buildId":"lCx8W2dIrIX8zJLeGMmLd","assetPrefix":"/ISBRA","nextExport":true,"autoExport":true,"isFallback":false,"scriptLoader":[]}</script></body></html>